### PR TITLE
docs: builder,runtime images

### PR DESCRIPTION
**Pull request by kkourt**
Add a paragraph about what the expectation is before starting the
process. Also, add a comment on backporting about how to backport these
changes.

Signed-off-by: Kornilios Kourtis <kornilios@isovalent.com>

### CODE REVIEW

File: Documentation/contributing/development/images.rst [hunk 1]
```diff
@@ -190,9 +190,23 @@ Image dependency:
...
+versions, build the necessary images and update all the appropriate
+locations in the Cilium codebase. Hence, before executing the following steps,
+the user should have such a commit (e.g., see
+`this commit
+<https://github.com/cilium/cilium/pull/17713/commits/b7a37ff80df8681d25a24fd5b464082d360fc6e2>`__)
+in their local tree. After following the steps below, the result would be another
+commit with the image updates (e.g,. see `this commit
+<https://github.com/cilium/cilium/pull/17713/commits/bd3357704647117fa9ef4839b9f603cd0435b7cc>`__).
+Please keep the two commits separate to ease backporting.
+
```
Comment (joestringer) [inline Documentation/contributing/development/images.rst:209]: :+1: 

There's one more case that applies maybe more to me, but we could mention it here too.

```suggestion
Please keep the two commits separate to ease backporting.

If you only wish to update the packages in these images, then you can manually update the `FORCE_BUILD` variable in `images/runtime/Dockerfile` to have a different value and then proceed with the steps below.
```

**Review by joestringer — APPROVED** [review]
I have one more suggestion but LGTM

File: Documentation/contributing/development/images.rst (hunk 1 above)
Comment (kkourt) [reply Documentation/contributing/development/images.rst:209]: Added!

**Review by aditighag — APPROVED** [review]
Thanks! I rely on your experience following Joe's instructions. :)

**Pull request merged**